nomt/src/page_cache.rs

```diff
@@ -298,6 +298,21 @@ fn shard_regions(num_shards: usize) -> Vec<(PageRegion, usize)> {
     regions
 }
 
+// see comments in `shard_regions`
```

Comment:
Probably would be better to mention that this returns the indicies according to allocation in shard_regions